learnyounode/exercises/08-juggling-async.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+let bl = require('bl');
+let http = require('http');
+let result = [];
+let count = 0;
+
+let print = function() {
+  for (let i = 0; i < 3; i++) {
+    console.log(result[i]);
+  }
+}
+
+function httpGet(i) {
+  http.get(process.argv[2 + i], function (response) {
+    response.pipe(bl(function (err, data) {
+      result[i] = data.toString();
+      count++;
+
+      if (count === 3) {
+        print();
+      }
+    }))
+  })
+}
+
+for (let i = 0; i < 3; i++) {
+  httpGet(i)
+}
```

Comment:
Nice job! Now as a challenge, what if we didn't know how many URLs we were going to get?